src/store/modules/keycodes/quantum.js

```diff
@@ -8,6 +8,34 @@ function makeLT(layer) {
     title: `kc on tap, switch to layer ${layer} while held`
   };
 }
+// make a One-Shot Mod Keycode Definition
+const osmLookup = {
+    'LSft': ['LSft' , 'Left Shift'],
+    'LCtl': ['LCtl' , 'Left Control'],
+    'LAlt': ['LAlt' , 'Left Alt'],
+    'LGUI': ['LGUI' , 'Left GUI'],
+    'RSft': ['RSft' , 'Right Shift'],
+    'RCtl': ['RCtl' , 'Right Control'],
+    'RAlt': ['RAlt' , 'Right Alt'],
+    'RGUI': ['RGUI' , 'Right GUI'],
+    'Hypr': ['Hyper', 'Left Control, Shift, Alt and GUI'],
+    'Meh' : ['Meh'  , 'Left Control, Shift, and Alt']
+}
+function makeOSM(mod) {
+  // visual key legend
+  let text = osmLookup[mod][0];
```

Comment:
this will break if mod is undefined

Reply:
I never actually figured out what to do about this.

Reply:
so typically you guard against mod not being a valid value. checking that it resolves first, so that the 2nd index lookup doesn't fail. It's not a huge deal because this is used internally. Just be aware it could crash if someone feeds it bad data in an update.

Reply:
Yeah, I get what the problem is. I just couldn't think of a way to check that the values are valid.